### PR TITLE
update VAOS UI when API returns no confirmed and pending appts

### PR DESCRIPTION
**Pull request by jenniemc**
## Description
update Confirmed and Pending list page when no such appointments are returned

## Testing done
local unit test

## Screenshots
image

image

## Acceptance criteria
- [ ] Text description stating there's no confirmed and pending appointments

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/containers/ConfirmedAppointmentListPage.jsx [hunk 1]
```diff
@@ -26,22 +26,51 @@ export class ConfirmedAppointmentListPage extends React.Component {
...
-              )}
+              {status === FETCH_STATUS.succeeded &&
+                appointments.length === 0 && (
+                  <div className="vads-u-margin-bottom--2 vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-bottom--3">
+                    <h2 className="vads-u-margin--0 vads-u-margin-bottom--2p5 vads-u-font-size--md">
+                      You don't have any confirmed appointments at this time.
+                    </h2>
+                    <p>
+                      You can schedule an appointment now, or you can call your{' '}
+                      <a href="/find-locations">VA Medical center</a> to
```
Comment (jbalboni) [inline src/applications/vaos/containers/ConfirmedAppointmentListPage.jsx]: I think we should open this link in a new tab. You'd need to update it to `<a href="/find-locations" target="_blank" rel="noopener noreferrer">` to do so. Should do the same thing in the pending appointment page, too.

**Review by jbalboni — COMMENTED** [review]
One minor comment, but looks great otherwise.

**Comment by jenniemc** [comment]
@jbalboni, just pushed the new commit

**Review by jbalboni — APPROVED** [review]

**Review by jhonny218 — COMMENTED** [review]
@department-of-veterans-affairs/frontend-review-group I have reviewed it and it looks good

**Review by rianfowler — APPROVED** [review]

**Pull request merged**